subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/android/AndroidIncrementalExecutionPerformanceTest.groovy

```diff
@@ -53,7 +53,7 @@ class AndroidIncrementalExecutionPerformanceTest extends AbstractIncrementalExec
 
     def "abi change#configurationCaching"() {
         given:
-        if (configurationCachingEnabled) {
+        if (configurationCachingEnabled && IncrementalAndroidTestProject.SANTA_TRACKER == testProject) {
```

Comment:
❓ Why it's limited only to Santa Tracker?

Reply:
The workaround is only needed with SantaTracker